src/components/ec-blog-item/ec-blog-item.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <div
+    class="ec-blog-item"
+  >
+    <div
+      class="tw-grid tw-border-b tw-border-solid tw-border-gray-6"
```

Comment:
could we have these as part of `.ec-blog-item` with `@apply`?

Reply:
As Carlos say you can remove this `div` and put all these classes on the `.ec-blog-item` with the `@apply`